pkg/kubelet/server/metrics/metrics.go

```diff
@@ -40,7 +40,7 @@ var (
 		// server_type aims to differentiate the readonly server and the readwrite server.
 		// long_running marks whether the request is long-running or not.
 		// Currently, long-running requests include exec/attach/portforward/debug.
-		[]string{"method", "path", "server_type", "long_running"},
+		[]string{"method", "path", "server_type", "long_running", "code"},
 	)
 	// HTTPRequestsDuration tracks the duration in seconds to serve http requests.
```

Comment:
Should we be adding the code label to any of the metrics below?

Reply:
I think `HTTPRequestsDuration` metrics  can add, while `HTTPInflightRequests` no neccessary. 
How about ? @tallclair